sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/dot/ToDotNodesVisitor.java

```diff
@@ -145,6 +160,14 @@ String getFunction(DBSPSimpleOperator node) {
     }
 
     String getColor(DBSPSimpleOperator operator) {
+        /*
+        // Show in green projection operators
```

Comment:
Remove?

Reply:
no, these are useful for future debugging sessions